backend.native/tests/codegen/initializers/sharedVarInInitBlock_main.kt

```diff
@@ -0,0 +1 @@
+fun main() = println(A().x)
```

Comment:
Please prefer making tests self-containing. Using `assertEquals` should help.

Reply:
Does it work with link tests? I remember some issues here...

Reply:
It should.

Reply:
Reports error: `error: could not find 'main' in '<root>' package.`

Reply:
Ah, ok, worked after explicitly adding `-tr` flag to a compilation.

Reply:
This probably just workarounds the original issue

Reply:
https://github.com/JetBrains/kotlin-native/commit/cd6829b25a6162f958b164dc544de7a9826289e7 ?